src/lib/y2storage/enum_mappings.rb

```diff
@@ -98,5 +98,14 @@ module EnumMappings
         "twofish256_old" => ::Storage::EncryptionType_TWOFISH256_OLD,
         "luks"           => ::Storage::EncryptionType_LUKS
       }
+
+    MOUNT_BY_TYPES =
+      {
+        "device" => Storage::MountByType_DEVICE,
```

Comment:
The other definitions above use fully qualified `::Storage` name space, this looks inconsistent.